deploy/018-native-token-gateway.ts

```diff
@@ -0,0 +1,73 @@
+import { ethers } from "hardhat";
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+
+import { contracts as ilBscMainnet } from "../deployments/bscmainnet.json";
+import { contracts as ilBscTestnet } from "../deployments/bsctestnet.json";
+import { contracts as ilEthereum } from "../deployments/ethereum.json";
+import { contracts as ilSepolia } from "../deployments/sepolia.json";
+import { getConfig } from "../helpers/deploymentConfig";
+
+interface VTokenConfig {
+  name: string;
+  address: string;
+}
+
+const VWNativeInfo: { [key: string]: VTokenConfig } = {
+  bsctestnet: {
+    name: "vWBNB_LiquidStakedBNB",
+    address: ilBscTestnet.VToken_vWBNB_LiquidStakedBNB.address,
+  },
+  bscmainnet: {
+    name: "vWBNB_LiquidStakedBNB",
+    address: ilBscMainnet.VToken_vWBNB_LiquidStakedBNB.address,
+  },
+  sepolia: {
+    name: "vWETH_Core",
+    address: ilSepolia.VToken_vWETH_Core.address,
+  },
```

Comment:
1. If we want to include the LST pool too, then i think we can store an array of markets address and name for each network.
And then iterate over the array to deploy the gateway contract for each market at once. Wdyt @chechu ?
2. We will need to change the underlying of the LST pool vWETH too as it contains the mock tokens as underlying for now.
3. For the second point we will need to create a vip and also include the commands to change the mappings related to the token in pool registry.

Reply:
1. LGTM. I think the address would be enough because the symbol can be recovered from the blockchain dynamically. But if we need the symbol too, it works for me
2. We have updated the underlying token of the vWETH market in the LST pool ([TX](https://sepolia.etherscan.io/tx/0xa28c6a724567572426126139c9cc857ae0385c9e441ae93235001356911b570f))
3. We could add the needed commands in the [open PR](https://github.com/VenusProtocol/vips/pull/195). wdyt?

Reply:
Refactored the deployment script: https://github.com/VenusProtocol/isolated-pools/pull/361/commits/d0085e2312b1d048a394e5cf57385cbf65aa76e9

For third point I think we can make a new vip for the this LST pool vWETH market **NativeTokenGateway** contract in the same PR as mentioned.